### DISCUSSION TITLE
I redirected the USB flash drive, but I don't see any effect

### DISCUSSION THREAD

**Discussion by LLeext** [FreeRDP/FreeRDP#9330] [Q&A]
I redirected the USB flash drive, but I don't see any effect
`wfreerdp.exe /u:lwt /p:lwtlwt123 /v:192.168.16.132 /usb:id,13FE:6300`
![image](https://github.com/FreeRDP/FreeRDP/assets/129752288/e09c1b8e-40a3-4831-9ef9-3337e824ea32)

**Comment by akallabeth** [comment]
well, nothing we can do here ;)
your remote end (`xrdp` in that case?) must support low level `USB` redirection too.
but I think what you are actually looking for is drive redirection (`/drive:name,/mount/path/of/your/drive`)
